src/main/java/uk/gov/hmcts/divorce/caseworker/service/task/SendApplicationIssueNotifications.java

```diff
@@ -37,7 +37,10 @@
                 }
             } else {
                 applicationIssuedNotification.sendToJointApplicant1(caseData, caseId);
-                applicationIssuedNotification.sendToJointApplicant2(caseData, caseId);
+                if (Objects.nonNull(caseData.getApplication().getApplicant1KnowsApplicant2EmailAddress())
```

Comment:
do we need a test to ensure when this is null `applicationIssuedNotification.sendToJointApplicant2` is not executed ?

Reply:
@nitinprabhu - yes I did try that but due to the fact that the same mock is used for the other email execution I couldn't use verifyNoInteractions